source/cursorManager.py

```diff
@@ -297,6 +297,9 @@ def _selectionMovementScriptHelper(self,unit=None,direction=None,toPosition=None
 				# 1. Caret at 1; selection (1, 1)
 				# 2. Shift+rightArrow: selection (1, 2)
 				newInfo.setEndPoint(oldInfo, "startToStart")
+		# _isSelectionAnchoredAtStart is not the same as _lastSelectionMovedStart,
+		# i.e. their default values are opposite and _isSelectionAnchoredAtStart is only updated if the selection has changed.
+		self._isSelectionAnchoredAtStart = not self._lastSelectionMovedStart

```

Comment:
I get that they're reversed, but I don't follow the bit about it only being updated if the selection changes. Would it really matter if isSelectionAnchoredAtStart just always returned the reverse of _lastSelectionMovedStart? I'm thinking we should get rid of _lastSelectionMovedStart and just flip everything to use the new attribute.